src/main/java/de/hs_mannheim/informatik/ct/util/ScheduledMaintenanceTasks.java

```diff
@@ -41,9 +41,10 @@
     @Autowired
     private EventVisitService eventVisitService;
 
+    public static final String FORCED_END_TIME = "00:00:00";
+
     private final int CRON_HOUR = 3;
     private final int CRON_MINUTE = 55;
```

Comment:
it might make sense, to set those variables as environement variables, wouldn't it?